src/main/controllers/service/ServiceSearchResultsController.ts

```diff
@@ -0,0 +1,41 @@
+import { Response } from 'express';
+import { FactRequest } from '../../interfaces/FactRequest';
+import autobind from 'autobind-decorator';
+import { FactApi } from '../../utils/FactApi';
+import { cloneDeep } from 'lodash';
+import { ServiceSearchResults } from '../../interfaces/ServiceSearchResults';
+
+@autobind
+export class ServiceSearchResultsController {
+
+  constructor(
+    private readonly api: FactApi
+  ) { }
+
+  public async get(req: FactRequest, res: Response) {
+    enum Catchment {
+      National = 'national'
+    }
+    const serviceAreaData = await this.api.getServiceArea(req.params.serviceArea);
+    const courtsData = serviceAreaData.serviceAreaCourts;
+
+    const data: ServiceSearchResults = {
+      ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['service-results'])
+    };
+    for( const court of courtsData ){
+      if( court.catchmentType === Catchment.National ){
+        data.nameOfCourt = data.nameOfCourt
```

Comment:
i dont see the point in this, if you were replacing a specific word in a sentence but as its the whole word you want, why not pass it with the data like we do before and in the html use court.name.